shared/helpers/view.js

```diff
@@ -4,12 +4,12 @@
 
 var _ = require('underscore'),
     getProperty = require('../../lib/getProperty'),
-    isServer = typeof window === 'undefined',
     BaseView;
 
 module.exports = function (Handlebars) {
   return function (viewName, options) {
-    var ViewClass, html, viewOptions, view, app;
+    var isServer = typeof window === 'undefined',
```

Comment:
I moved this here so that I could stub out `window` in the specs. Open to other suggestions if we don't want it declared as part of the function